src/components/IconButton/IconButton.e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { browser } from 'protractor';
```

Comment:
Since we don't have any special interactions we need to test, we can remove this file